src/interactive-window/debugger/startupCodeProvider.ts

```diff
@@ -32,6 +32,11 @@ export class InteractiveWindowDebuggingStartupCodeProvider implements IStartupCo
         }
 
         if (kernel.notebook?.notebookType === InteractiveWindowView) {
+            if (!isLocalConnection(kernel.kernelConnectionMetadata)) {
+                // With remove kernel connection in the web, we use the new approach, i.e. Jupyter debugger protocol.
```

Comment:
This was missing in the refactoring that was done here https://github.com/microsoft/vscode-jupyter/pull/10853